### PR TITLE
chore: Change date and release notes for 10.5.0

### PR DESCRIPTION
**Pull request by johanandren**
References #4231

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@ Additional Use Grant:
     Connecting to a Play Framework websocket and/or Play Framework
     request/response bodies for server and play-ws client.
 
-Change Date:          TBD ( 3 years after 10.5.0 release )
+Change Date:          2026-02-21 ( 3 years after 10.5.0 release )
```
Comment (patriknw) [inline LICENSE.txt:18]: I think the original instruction was to just have the date here, see https://github.com/akka/akka/commit/ca7052179650aec65ef9f5e73276f79be6542263

File: LICENSE.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@ Business Source License 1.1
 Parameters
 
 Licensor:             Lightbend, Inc.
-Licensed Work:        Akka HTTP 10.5.x
+Licensed Work:        Akka HTTP 10.5.0
```
Comment (patriknw) [inline LICENSE.txt:6]: 👍

**Comment by patriknw** [comment]
here is one more place https://github.com/akka/akka-http/blob/main/build.sbt#L32
it will end up in the pom.xml
Didn't we automate that somewhere else, taking the tag from the build?

File: LICENSE.txt (hunk 1 above)
Comment (johanandren) [reply LICENSE.txt]: 👍

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**